docs/learning-powershell/README.md

```diff
@@ -113,6 +113,7 @@ Recommended Training and Reading
 - [PowerShell Quick Reference Guides][quick-reference] by PowerShellMagazine.com
 - [PowerShell 5 How-To Videos][script-guy-how-to] by Ed Wilson
 - [PowerShell Documentation](https://docs.microsoft.com/powershell)
+- [Learn Windows PowerShell in a Month of Lunches][month-of-lunches]
```

Comment:
Should it be in Commercial Resources?

Reply:
I agree, can I move it without performing a new pull request?